addons/mail/tests/discuss/test_rtc.py

```diff
@@ -108,8 +110,7 @@ def test_10_start_call_in_chat_should_invite_all_members_to_call(self):
             [
                 (self.cr.dbname, 'discuss.channel', channel.id),  # update new session
                 (self.cr.dbname, 'discuss.channel', channel.id),  # message_post "started a live conference" (not asserted below)
-                (self.cr.dbname, 'res.partner', self.user_employee.partner_id.id),  # update of last interest (not asserted below)
-                (self.cr.dbname, 'res.partner', test_user.partner_id.id),  # update of last interest (not asserted below)
```

Comment:
Note: adaptations from change in code to send this value on channel bus rather than each member, to fix performance.
The same value was sent to everyone.